src/main/java/org/generationcp/middleware/pojos/workbench/WorkbenchUser.java

```diff
@@ -66,21 +65,89 @@
 
 	private static final long serialVersionUID = 1L;
 
+	public static final String GET_USERS_BY_PROJECT_ID = "SELECT  "
+		+ "  users.userid, "
+		+ "  users.instalid, "
+		+ "  users.ustatus, "
+		+ "  users.uaccess, "
+		+ "  users.utype, "
+		+ "  users.uname, "
+		+ "  users.upswd, "
+		+ "  users.personid, "
+		+ "  users.adate, "
+		+ "  users.cdate, "
+		+ "  pr.fname, "
+		+ "  pr.lname "
+		+ "    FROM "
+		+ "       workbench_project p "
+		+ "           INNER JOIN "
+		+ "       workbench_crop wc ON p.crop_type = wc.crop_name "
+		+ "           INNER JOIN "
+		+ "       crop_persons cp ON cp.crop_name = wc.crop_name "
+		+ "           INNER JOIN "
+		+ "       users ON cp.personid = users.personid "
+		+ "           INNER JOIN "
+		+ "       persons pr ON pr.personid = users.personid "
+		+ "           INNER JOIN "
+		+ "       users_roles ur ON ur.userid = users.userid "
+		+ "           INNER JOIN role r ON ur.role_id = r.id  "
+		+ "   where  (r.role_type_id =  " + RoleType.INSTANCE.getId()
+		+ "     or (r.role_type_id = " + RoleType.CROP.getId() + " and ur.crop_name = :cropName)  "
+		+ "     or (r.role_type_id =  " + RoleType.PROGRAM.getId()
+		+ " and ur.crop_name = :cropName AND ur.workbench_project_id = p.project_id))  "
+		+ "    AND "
+		+ "       p.project_id = :projectId "
+		+ "    GROUP BY users.userid";
+
+	public static final String GET_ACTIVE_USER_IDS_BY_PROJECT_ID =
```

Comment:
Are we really interested in just users with program role type? If so, we should rename the name of this query to reflect so.